weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/TraceUsageStats.tsx

```diff
@@ -72,41 +111,46 @@ export const TraceUsageStats = ({
   );
 };
 
-export const getUsageFromCellParams = (params: {[key: string]: any}) => {
-  const usage: {[key: string]: UsageData} = {};
+export const getCostFromCellParams = (params: {[key: string]: any}) => {
```

Comment:
We are starting to add some expected structure to `TraceCallSchema`. I think the typing contracts might be better if we modify `TraceCallSchema` in a way like:

```
# Move CostData def here

type WeaveComputedSummaryType = {
  "cost"?: CostData,
  // Other computed summary fields can be added here
}

type SummaryKeyedDictType = KeyedDictType & {
  "weave"?: WeaveComputedSummaryType;
}

export type TraceCallSchema = {
  # <snip>
  summary?: SummaryKeyedDictType;
  # <snip>
};
```

The more clearly defines the expectation from the server inside the the server interface. Then, if we decide to change the server output, we have a single type definition to change in an authoritative location. 

Furthermore, your functions like `getCostFromCellParams` that are specific to rendering should operate on `WeaveComputedSummaryType`, rather than a free-form dict.

Reply:
Note: note required for this PR, but this pattern can also be used for other computed fields we will soon add as well as the `weave` payload in attributes

Reply:
i am pretty sure you can also declare the pyandtic type on the server like:

```
class KeyedDictType(BaseModel):
    class Config:
        extra = "allow"

class SummaryKeyedDictType(KeyedDictType):
    weave: ...
```

this would be really nice to formalize the representation so we can reason about it in the future